libkpod/wait.go

```diff
@@ -0,0 +1,43 @@
+package libkpod
+
+import (
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+
+func isStopped(c *ContainerServer, ctr *oci.Container) bool{
+	c.runtime.UpdateStatus(ctr)
```

Comment:
This can return an error, I think? And the function you pass to PollImmediateInfinite can return an error too, so you can bubble it up